### PR TITLE
fix: update css for entity renderer text to inherit font properties

### PR DESCRIPTION
**Pull request by arjunlalb**
## Description
update css for entity renderer text to inherit font properties

### Testing
Manually tested
### Checklist:
- [x] My changes generate no new warnings
- [NA ] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/observability/src/shared/components/entity-renderer/entity-renderer.component.scss [hunk 1]
```diff
@@ -2,11 +2,13 @@
 @import 'font';
 
 .ht-entity-renderer {
+  @include body-1-regular();
```
Comment (aaron-steinfeld) [inline projects/observability/src/shared/components/entity-renderer/entity-renderer.component.scss:5]: Why declare the font if we're also trying to inherit it?

Comment (arjunlalb) [reply projects/observability/src/shared/components/entity-renderer/entity-renderer.component.scss:5]: for the defaults that are not overridden.

Comment (aaron-steinfeld) [reply projects/observability/src/shared/components/entity-renderer/entity-renderer.component.scss:5]: Is that anything beyond color? If it's not, can we expose a variable like default-body-color (and update the font mixins to something like)
```scss
$default-body-color:  $gray-9;
@mixin body-1-regular($color: $default-body-color) {
```
and this to just declare the color? I think it's confusing and prone to breaking to use a mixin for one property, and override the others intentionally.

Comment (arjunlalb) [reply projects/observability/src/shared/components/entity-renderer/entity-renderer.component.scss:5]: Anand is revamping the views. Dropping this PR.

**Pull request closed without merging**